<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система тестирования | МГТУ им. Н.Э. Баумана</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #FFFFFF;
}
#left_panel {
    background-color: #003366;
    border-right: 1px solid #002244;
}
#right_panel {
    background-color: #FFFFFF;
    padding: 40px;
}
QLabel {
    color: #333333;
    font-size: 16px;
    font-weight: bold;
    font-family: "Segoe UI", sans-serif;
    margin-bottom: 8px;
    padding: 5px;
}
QLabel#surname_label, QLabel#name_label, QLabel#group_label {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    margin-bottom: 8px;
}
#left_panel QLabel {
    color: #FFFFFF;
    margin: 0;
}
QLineEdit {
    font-size: 14px;
    padding: 8px 12px;
    border: 1px solid #CCCCCC;
    border-radius: 4px;
    background-color: #FFFFFF;
    font-family: "Segoe UI", sans-serif;
    min-height: 35px;
    margin-bottom: 20px;
    text-align: left;
    width: 100%;
}
QLineEdit:focus {
    border: 1px solid #2196F3;
    background-color: #E3F2FD;
}
QLineEdit::placeholder {
    color: #999999;
    font-size: 14px;
    font-family: "Segoe UI", sans-serif;
}
QPushButton#login_button {
    font-size: 16px;
    font-weight: bold;
    padding: 0;
    border-radius: 4px;
    background-color: #2196F3;
    color: white;
    border: none;
    min-height: 40px;
    margin-top: 25px;
    font-family: "Segoe UI", sans-serif;
}
QPushButton#login_button:hover {
    background-color: #1976D2;
}
QPushButton#teacher_button {
    font-size: 14px;
    padding: 0;
    border-radius: 4px;
    background-color: transparent;
    color: #2196F3;
    border: 1px solid #2196F3;
    min-height: 40px;
    margin-top: 10px;
    font-family: "Segoe UI", sans-serif;
}
QPushButton#teacher_button:hover {
    background-color: #E3F2FD;
}
QPushButton#export_pdf_button {
    font-size: 12px;
    padding: 4px 8px;
    border-radius: 4px;
    background-color: #4CAF50;
    color: white;
    border: none;
    min-height: 25px;
    max-width: 120px;
    margin: 5px;
    font-family: "Segoe UI", sans-serif;
}
QPushButton#export_pdf_button:hover {
    background-color: #388E3C;
}
#error_message {
    color: #DC3545;
    font-size: 12px;
    margin-top: 4px;
    font-family: "Segoe UI", sans-serif;
}
#year_label {
    color: #666666;
    font-size: 14px;
    margin-top: 20px;
    padding: 10px;
    font-family: "Segoe UI", sans-serif;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="left_panel">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <spacer name="verticalSpacer_top">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="logo_label">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../resources/icons/bmstu_logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="title_label">
         <property name="text">
          <string>Система тестирования</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="subtitle_label">
         <property name="text">
          <string>МГТУ им. Н.Э. Баумана</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="description_label">
         <property name="text">
          <string>Лабораторный практикум
по электротехнике</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="right_panel">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="form_container">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>40</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>40</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="surname_label">
            <property name="text">
             <string>Фамилия</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="surname_input">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>35</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Введите фамилию</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="name_label">
            <property name="text">
             <string>Имя</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name_input">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>35</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Введите имя</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="group_label">
            <property name="text">
             <string>Группа</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="group_input">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>35</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Введите номер группы</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="export_pdf_button">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Экспорт в PDF</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="login_button">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Войти</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="teacher_button">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Вход для преподавателя</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="year_label">
            <property name="geometry">
             <rect>
              <x>650</x>
              <y>650</y>
              <width>120</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #666666;
margin-top: 10px;
padding: 5px;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignBottom</set>
            </property>
            <property name="text">
             <string>Текущий год: 2024</string>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
